core/src/org/pentaho/platform/engine/core/audit/MessageTypes.java

```diff
@@ -78,4 +78,6 @@
 
   public static String ACTION_SEQUENCE_EXECUTE_CONDITIONAL = "condition"; //$NON-NLS-1$
 
+  public static String CANCELLED = "cancelled"; //$NON-NLS-1$
+
```

Comment:
never have a non final public static variable. If it is a constant, make it final so that no one can change it. (Or the constant is not constant. :) )

If it is public, and not final, any one (including scripts running in a report) can change it to funny values and thus break the system in rather interesting ways.